src/utils/metrics.ts

```diff
@@ -190,6 +190,13 @@ export function createMetrics(registry: Registry, register = true) {
         registers
     })
 
+    const walletsProcessingTime = new Histogram({
+        name: "alto_executor_wallets_processing_time_seconds",
```

Comment:
```suggestion
        name: "alto_executor_wallets_processing_duration_seconds",
```

duration is a more common word for naming Histograms